1080i/Includes_OSD.xml

```diff
@@ -631,6 +631,18 @@
                 </control>
             </control>
 
+            <control type="image">
+                <bottom>150</bottom>
+                <right>view_pad</right>
+                <width>500</width>
+                <height>281</height>
+                <texture background="true">$VAR[Image_Player_Clearart]</texture>
+                <visible>![!String.IsEmpty(Window(Home).Property(OSDInfo)) | Player.ShowInfo | Window.IsActive(DialogFullScreenInfo.xml) | [Player.Paused + Skin.HasSetting(ShowInfoPaused)]]</visible>
```

Comment:
Clearart overlaps info when unpausing if the skin setting `Always display on-screen info` is enabled.
![Honeyview_screenshot430](https://user-images.githubusercontent.com/4618719/66708348-ec7ef300-ed9a-11e9-9717-2e6f835943fd.jpg)

The setting is "ShowInfoPaused" and you need to change the visibility condition to not show clearart if that setting is enabled. i.e. Change to:
`<visible>![!String.IsEmpty(Window(Home).Property(OSDInfo)) | Player.ShowInfo | Window.IsActive(DialogFullScreenInfo.xml) | Skin.HasSetting(ShowInfoPaused)]</visible>`